src/main/java/educationalproject/programmingstuff/servicies/dto/CommodityItemResponseDto.java

```diff
@@ -0,0 +1,23 @@
+package educationalproject.programmingstuff.servicies.dto;
+
+import educationalproject.programmingstuff.model.Item;
+import lombok.Builder;
+import lombok.Value;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Value
+@Builder
+public class CommodityItemResponseDto {
+
+    @NotNull(message = "Id can't be null")
+    long id;
```

Comment:
if it's a primitive type, then @NotNull is useless

Reply:
Done